compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -3774,72 +3749,36 @@ static void lookupScheme2(TR::Node *node, bool unbalanced, bool fromTableEval, T
 
    int32_t   preInt = fromTableEval?0:node->getChild(2)->getCaseConstant();
 
-   if (isInt64)
-      {
-      if (two_reg)
-         {
-         loadConstant(cg, node, (int32_t)(preInt >> 32), valRegister->getHighOrder());
-         loadConstant(cg, node, (int32_t)preInt, valRegister->getLowOrder());
-         }
-      else
-         loadConstant(cg, node, preInt, valRegister);
-      }
-   else
-      {
-      loadConstant(cg, node, (int32_t) preInt, valRegister);
-      }
+   loadConstant(cg, node, preInt, valRegister);
 
    for (int i=2; i<total; i++)
       {
       TR::Node *child = node->getChild(i);
 
-      if (isInt64)
-         {
-         if (cg->comp()->target().is64Bit())
-            {
-            generateTrg1Src2Instruction(cg, TR::InstOpCode::cmp8, node, cndRegister, selector, valRegister);
-            }
-         else
-            {
-            generateTrg1Src2Instruction(cg, TR::InstOpCode::cmp4, node, cndRegister, selector->getHighOrder(), valRegister->getHighOrder());
-            generateConditionalBranchInstruction(cg, TR::InstOpCode::bne, node, toDefaultLabel, cndRegister);
-            generateTrg1Src2Instruction(cg, TR::InstOpCode::cmp4, node, cndRegister, selector->getLowOrder(), valRegister->getLowOrder());
-            }
-         }
-      else
-         {
-         generateTrg1Src2Instruction(cg, TR::InstOpCode::cmp4, node, cndRegister, selector, valRegister);
-         }
+      generateTrg1Src2Instruction(cg, TR::InstOpCode::cmp4, node, cndRegister, selector, valRegister);
 
       if (unbalanced)
-	 {
+	      {
          bcond = conditions;
          if (child->getNumChildren() > 0)
-	    {
+	         {
             cg->evaluate(child->getFirstChild());
             bcond = bcond->clone(cg, generateRegisterDependencyConditions(cg, child->getFirstChild(), 0));
-	    }
+	         }
          generateDepConditionalBranchInstruction(cg, TR::InstOpCode::beq, node, child->getBranchDestination()->getNode()->getLabel(), cndRegister, bcond);
-	 }
+	      }
       else
-	 {
+	      {
          generateConditionalBranchInstruction(cg, TR::InstOpCode::beq, node, child->getBranchDestination()->getNode()->getLabel(), cndRegister);
-	 }
+	      }
       if (i<total-1)
-	 {
+	      {
```

Comment:
It looks like somebody had used tabs to indent some of these lines.  May I ask you to change them to use spaces for indentation?  While you're at it, may I ask you to change other uses of tabs in this file to spaces?